packages/core-js/package.json

```diff
@@ -9,5 +9,10 @@
   "keywords": [],
   "author": "",
   "license": "MIT",
-  "devDependencies": {}
+  "devDependencies": {},
+  "dependencies": {
+    "chalk": "4.1.2",
+    "inquirer": "8.1.2",
+    "mustache": "4.2.0"
```

Comment:
Isn't rendering a template a very deployer plugin specific task? Where do you see anyone using templating from our utils for something else?

Reply:
yep, it's super deployer specific... I'll pull it back